src/components/molecules/MapCell/MapCell.tsx

```diff
@@ -53,3 +53,5 @@ export const MapRow: React.FC<MapRowProps> = ({
     </div>
   );
 };
+
+export const MapCell = React.memo(_MapCell);
```

Comment:
@mike-lvov what was the intended purpose of this change? My understanding is that `MapCell` (formerly `MapRow`) is only ever used in `src/components/templates/PartyMap/components/Map/hooks/useMapGrid.tsx`, within a section that is already `useMemo`'d. So it would seem that this change isn't actually going to do anything to improve the performance?